src/App.tsx

```diff
@@ -15,48 +17,80 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState<SortType>(SortType.NONE);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const sortedList = getReorderedGoods(
+    goodsFromServer,
+    sortType,
+    isReversed,
+  );
+
+  const resetCondition = sortType !== SortType.NONE || isReversed;
```

Comment:
Use more descriptive names
```suggestion
  const isResetButtonActive = sortType !== SortType.NONE || isReversed;
```